src/core/MeshBVH.js

```diff
@@ -875,13 +881,31 @@ export class MeshBVH {
 		trianglePool.releasePrimitive( triangle );
 		trianglePool.releasePrimitive( triangle2 );
 
-		return closestDistance;
+		if ( target2 ) {
 
-	}
+			if ( ! target2.point ) target2.point = tempTargetDest2.clone();
+			else target2.point.copy( tempTargetDest2 );
+			target2.distance = closestDistance,
+			target2.faceIndex = closestDistanceOtherTriIndex;
```

Comment:
Yes, I think ```target2.point``` should be converted to the other geometry frame. But the distance should remain the same, no need to recompute it?
Sorry for not spotting the frame issue.

Reply:
> But the distance should remain the same, no need to recompute it?

If the geometries are scaled differently to one another then the distance represented in each local frame would be different. Ie if "otherGeometry" were scaled to `0.1` while the current geometry were scaled to `10` then there's a scale difference of 100 between them meaning the distance would be different a factor of 100.

Not all transforms will have uniforms scales, though. It would probably be easiest to transform both points into the second geometry in order to recompute the distance.

> Sorry for not spotting the frame issue.

Ha I've been working on this project for awhile now and there are still things I'm surprised I didn't see previously!